sql/5_0_2-to-6_0_0_upgrade.sql

```diff
@@ -1936,3 +1936,14 @@ INSERT INTO list_options(list_id,option_id,title,seq) VALUES ('condition-verific
 INSERT INTO list_options(list_id,option_id,title,seq) VALUES ('condition-verification', 'refuted', 'Refuted', 30);
 INSERT INTO list_options(list_id,option_id,title,seq) VALUES ('condition-verification', 'entered-in-error', 'Entered in Error', 40);
 #EndIf
+
+#IfMissingColumn procedure_order uuid
+ALTER TABLE `procedure_order` ADD `uuid` binary(16) DEFAULT NULL;
+#EndIf
+
+#IfUuidNeedUpdate procedure_order
```

Comment:
looking forward to when this looks something like:
```
#IfUuidNeedUpdateId procedure_order procedure_order_id
```
:)

Reply:
Wait is Over. Pushed the fix :)